demo/src/screens/foundationScreens/DarkModeScreen.tsx

```diff
@@ -1,22 +1,46 @@
 import React, {Component} from 'react';
 import {StyleSheet} from 'react-native';
-import {View, Text, Constants} from 'react-native-ui-lib';
+import {View, Text, Colors, Constants, SegmentedControl, SegmentedControlItemProps} from 'react-native-ui-lib';
```

Comment:
`SegmentedControlItemProps` is imported but not used

Reply:
Fixed